README.md

```diff
@@ -14,8 +15,11 @@ Sample senders and receivers to be used with the Switchboard project. https://gi
 2. Change directory to src  
 `cd src`
 
-3. Run the video receiver  
+3. Edit the configuration file if necessary  
+`vi config.json`
```

Comment:
We should include an extra blurb about how
- you can edit the config file to choose which camera the sender will use
- the ffmpeg command to use to get the name of the cameras you have
basically add the explanation in this PR to the readme.

Reply:
Agreed, done